backend/src/models/User.ts

```diff
@@ -5,43 +5,41 @@ const userSchema = new Schema(
     _id: {
       // ID generated by Google Oauth, must be called _id to be queried with .findById()
       type: String,
+      unique: true,
     },
     email: {
       type: String,
       required: true,
+      unique: true,
     },
-    //Pass in from google
-    firstName: {
+    // Pass in from google
+    name: {
       type: String,
       required: true,
     },
-    lastName: {
+    type: {
       type: String,
-      required: true,
-    },
-    isAlumni: {
-      type: Boolean,
-      default: false,
+      default: "Student",
     },
     linkedIn: {
       type: String,
     },
     phoneNumber: {
       type: String,
     },
-    //Only for students
+    // Only for students
     major: {
       type: String,
     },
-    //Only for students
+    // Only for students
     classLevel: {
       type: String,
     },
-    //Only for alumni
+    // Only for alumni
     company: {
-      type: Object,
+      type: String,
     },
```

Comment:
Company should be of type `ObjectId` and populated as a company object when fetched from mongoDB, I believe you can directly populate using `.populate()` when getting the user.